modules/pretix/default.nix

```diff
@@ -14,9 +14,21 @@ in
     host = mkOption {
       type = types.str;
       default = null;
-      example = "tickets.myhostname.com";
+      example = "demo.megaclan3000.de";
       description = "Host serving pretix web service";
     };
+    acmeMail = mkOption {
+      type = types.str;
+      default = null;
+      example = "pretix-admin@zugvoegelfestival.org";
+      description = "Email for SSL Certificate Renewal";
+    };
+    ticketMail = mkOption {
+      type = types.str;
+      default = null;
+      example = "no-reply@zugvoegelfestival.org";
+      description = "From Email from which tickets will be send.";
```

Comment:
If these options have no default value, you should set them in https://github.com/zugvoegel-festival/ticketing/blob/6fdef5a64fc7d4d9faccc196504dd3647280b68b/configuration.nix#L14

Reply:
done
[
](https://github.com/zugvoegel-festival/ticketing/blob/5c583f160d33cec0f2fe12c15e6ba2292983e9c1/configuration.nix#L16C1-L19C5)